events/galaxy-academy-2024.md

```diff
@@ -1,54 +1,124 @@
 ---
-layout: learning-pathway
+layout: event
 
 title: Galaxy Academy 2024
 description: |
-  The Galaxy Academy is a online training event for Beginners as well as learners who would like to improve there Galaxy data analysis skills. Over the course of one week we will have a different topic and focus every day. 
-tags: [beginner]
+  The Galaxy Academy is a online training event for Beginners as well as learners who would like to improve their Galaxy data analysis skills. Over the course of one week, we will have a different topic and focus every day. 
+
 draft: true
 
+date_start: 2024-10-07
+date_end: 2024-10-11
+
+cost: free 
+audience: Everyone who would like to get to know Galaxy, learn bioinformatics data analysis, or master a specific new kind of analysis is welcome.
```

Comment:
Suggestion : audience: Everyone who would like to get to know Galaxy and learn bioinformatics data analysis, whether you want to master a specific analysis or learn a new skill.

Reply:
Thanks